examples/with-schedules/main.tf

```diff
@@ -0,0 +1,79 @@
+provider "aws" {
+  region = "ap-southeast-1"
+
+  # Make it faster by skipping something
+  skip_metadata_api_check     = true
+  skip_region_validation      = true
+  skip_credentials_validation = true
+  skip_requesting_account_id  = true
+}
+
+module "eventbridge" {
+  source = "../../"
+
+  create_bus = false
+
+  trusted_entities = ["scheduler.amazonaws.com"]
+
+  attach_lambda_policy = true
+  lambda_target_arns   = [module.lambda.lambda_function_arn]
```

Comment:
Weirdly enough, this error message was because [the EventBridge Scheduler service is not available in `ap-southeast-1` region](https://aws.amazon.com/about-aws/whats-new/2023/03/amazon-eventbridge-scheduler-18-regions/) we used in the example. It took me some time to find it out :)

Now I can finish this PR.